InvenTree/plugin/helpers.py

```diff
@@ -102,7 +102,7 @@ def handle_error(error, do_raise: bool = True, do_log: bool = True, log_name: st
 
 def get_entrypoints():
     """Returns list for entrypoints for InvenTree plugins."""
-    return entry_points().get('inventree_plugins', [])
+    return entry_points(group='inventree_plugins')
```

Comment:
This change is failing tests for 3.9 - either we'll need to check the python version or handle the excpetion.

Do you have a link to the change in 3.12 which breaks this behaviour?

Reply:
Yeah, it's a little hard to find, but here's the relevant bit (found [here](https://docs.python.org/3/library/importlib.metadata.html#entry-points)):

> The “selectable” entry points were introduced in importlib_metadata 3.6 and Python 3.10. Prior to those changes, entry_points accepted no parameters and always returned a dictionary of entry points, keyed by group. With importlib_metadata 5.0 and Python 3.12, entry_points always returns an EntryPoints object. See [backports.entry_points_selectable](https://pypi.org/project/backports.entry-points-selectable) for compatibility options.

Parsing that out, it looks like there's two options:

1. The most preferable option is to require `importlib_metadata >= 3.6`. This shouldn't require any changes to the actual code, as `entry_points()` will now behave the same across versions.
2. Alternatively, require `backports.entry-points-selectable` and use `from backports.entry_points_selectable import entry_points`.

Reply:
@SchrodingersGat I think we can resolve this now?